packages/resolver/src/adapters/unstoppable/api.ts

```diff
@@ -104,9 +108,19 @@ const UnstoppableApiAdaResponseSchema = z.object({
 })
 
 const UnstoppableApiGeneralResponseSchema = z.object({
+  meta: z.object({
+    blockchain: z.string(),
+  }),
   records: z.record(z.string(), z.string()),
 })
 
+const UnstoppableApiOtherBlockchainResponseSchema =
+  UnstoppableApiGeneralResponseSchema.refine(
+    ({records, meta}) =>
+      Object.keys(records).length > 0 ||
+      !['CARDANO', 'ADA'].includes(meta.blockchain),
```

Comment:
@banklesss I believe this is where the record of the resolver lives, i.e for Cardano domains, they are hosted on MATIC blockchain. Maybe the check is, if is not 404, but doesn't contain ...ADA... and there is a response 200, is for other blockchain.

Reply:
```curl -i -X GET \\n  'https://api.unstoppabledomains.com/resolve/domains/stackchain.blockchain' \\n  -H 'Authorization: Bearer ${ENV-TOKEN_UD}'
```

Reply:
i will take a look thanks

Reply:
yeah @stackchain i think we are refering to the same logic. I reflected that (if i am not wrong 😅)

please let me know if you have any suggestion

Reply:
```
Object.keys(records).length > 0 ||
      !['CARDANO', 'ADA'].includes(meta.blockchain),
```

if the records are not empty or "blockchain" is not empty and is not "ADA" or "CARDANO", the domain has assigned an address on another blockchain other than cardano.

Reply:
my point is `meta.blockchain` will never contain 'CARDANO' or 'ADA' cuz Cardano records are kept on MATIC network.

Reply:
ah! got it

Reply:
so @stackchain 

this response should be considered has "has not addresses in any blockchain" because doesn't have records right?

```
{
  meta: {
    domain: 'john.go',
    tokenId: '68339273160565029666262671917772397379555826087941337618091301383115593492574',
    namehash: '0x9716a86b68212d3124300dbeb22cbeb49f5131a8539bbdb747e3e4a2320c285e',
    blockchain: 'MATIC',
    networkId: 137,
    owner: '0x0c8ccc39309ddca7b803662231b780cf4131b6b1',
    resolver: '0xa9a6a3626993d487d2dbda3173cf58ca1a9d9e9f',
    registry: '0xa9a6a3626993d487d2dbda3173cf58ca1a9d9e9f',
    reverse: false,
    type: 'Uns'
  },
  records: {}
}
```

Reply:
if its right we can close the PR

Reply:
according to the official documentation records will contain the linked addresses

Reply:
I dunno, cuz it looks like the `records` property was added later, cuz for `john.go` I'm guessing the `owner` is the resolved addressed, and the resolver is the contract responsible to hold the data, needs checking.
Then the test for the `.length > 0` can be dropped, and the rule for returning the `WrongBlockchain` is -> the response is 200, and there is no ..ADA.. in the records.

Reply:
i will take a look

Reply:
mixing this and updating the tlds makes everything work as expected